pkgs/applications/version-management/git-and-tools/git-remote-hg/default.nix

```diff
@@ -1,36 +1,38 @@
-{ stdenv, lib, fetchFromGitHub, mercurial, makeWrapper
+{ stdenv, lib, fetchFromGitHub, python3Packages, makeWrapper
 , asciidoc, xmlto, docbook_xsl, docbook_xml_dtd_45, libxslt, libxml2
 }:
 
 stdenv.mkDerivation rec {
```

Comment:
Can't we use buildPythonApplication? This brings in wrapPython and does that postFixup thingy etc.